lib/default/htmlcsrc

```diff
@@ -22,6 +22,8 @@
     "lowercase-class-with-hyphen": true,    // true: ClassName should be lowercase words connected with hyphens
     "lowercase-id-with-hyphen": true,       // true: Id should be lowercase words connected with hyphens
     "nest": true,                           // true: Elements should be nested abiding by specific rules
+	"no-space-before-tag-end": true,        // true: There should be no whitespaces before tag end
+    "protocol-omitted-in-href": true,       // true: Protocol (http:// or https://) should be omitted from href attribute (there should be "//" instead)
```

Comment:
Maybe it will be better to use `false` as default value for these two rules, considering backward compatibility.